### ISSUE TITLE
Logging a 'null' string causes an exception.

### ISSUE THREAD

**Issue by feisley** [jongpie/NebulaLogger#256]
### Package Edition of Nebula Logger

Unlocked Package

### Package Version of Nebula Logger

v4.6.14

### New Bug Summary

We were logging code that could occasionally result in the string being passed to the logger being null. When this occurs the Logger itself emits an exception:

`System.NullPointerException: Attempt to de-reference a null object
`

The stack trace is:

```
Class.LogEntryEventBuilder.applyDataMaskRules: line 752, column 1
Class.LogEntryEventBuilder.setMessage: line 171, column 1
Class.Logger.debug: line 1570, column 1
```

The code that we had to produce this was of the form:

```
String results = someFunctionThatCanReturnNull();
Logger.debug(results); // Error thrown here
```

**Comment by jongpie** [comment]
Hi @feisley - thanks for reporting this! I thought I had already had this working a while ago, but it does indeed look like this is a bug - I think I just need to add an extra null check to avoid the error. I don't have a timeline for the next release yet, but I'll include a fix for this in the next release, `v4.6.15`.  

I'll let you know once i have this resolved. Let me know if you run into any other issues!

**Comment by jongpie** [comment]
Hey @feisley - this should now be resolved in the new release, [v4.6.15](https://github.com/jongpie/NebulaLogger/releases/tag/v4.6.15). Let me know if you come across any other issues, or have any other feedback. Thanks again for reporting this issue!
